src/utils/formatting.ts

```diff
@@ -73,9 +73,22 @@ export const liveChannelsURL = (playlistId: string, channelId?: string, play = f
   });
 };
 
-export const episodeURL = (episode: PlaylistItem, seriesId?: string, play: boolean = false, playlistId?: string | null) =>
-  addQueryParams(mediaURL(episode, playlistId, play), {
+export const episodeURL = ({
+  episode,
+  seriesId,
+  play,
+  playlistId,
+  mediaId,
+}: {
+  episode: PlaylistItem;
+  seriesId?: string;
+  play?: boolean;
+  playlistId?: string | null;
+  mediaId?: string | null;
+}) =>
+  addQueryParams(mediaURL(episode, playlistId, play || false), {
     seriesId,
+    mediaId,
```

Comment:
Why did you add mediaId as an additional query param?

Reply:
MediaSeries redirects to MediaSeriesEpisode where a media items is already an episode, so we lose initial media item which is series. We could use only `seriesId` and just use either a seriesId from custom params or mediaId but it will conflict with the case when a customer has `series` content type (new flow) and uses old flow.

Reply:
OK, let's keep the old flow for a while, but I still think we can use 1 seriesId. The media ID of interest should be on the episode, right?

Reply:
Fixed it.